packages/devtools_app/lib/src/config_specific/theme_overrides/theme_overrides_desktop.dart

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be found
+// in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+/// Environment-specific theme overrides, for example IDE-provided
+/// theming for embedded mode.
+class ThemeOverrides {
```

Comment:
move this class over to a shared location rather than duplicating it each conditional import.
Dart is really bad at type checking conditional imports so keeping the API imported conditionally small is important.

Reply:
I did this by reusing the class definition in `_stub.dart` and then just making a function that loads the IDE theme. If this isn't what you had in mind or I can do it better, let me know!